huaweicloud/services/iam/resource_huaweicloud_identity_user.go

```diff
@@ -159,9 +167,34 @@ func resourceIdentityUserCreate(ctx context.Context, d *schema.ResourceData, met
 	}
 
 	d.SetId(user.ID)
+
+	// update login protect
+	err = updateLoginProtect(iamClient, user.ID, d.Get("login_protect_verification_method").(string))
```

Comment:
as the login protect verification is disabled after the creation, we can call updateLoginProtect only when `login_protect_verification_method` is not empty.